src/GitHub.VisualStudio/Services/UsageTracker.cs

```diff
@@ -31,11 +30,13 @@ public sealed class UsageTracker : IUsageTracker, IDisposable
         public UsageTracker(
             IGitHubServiceProvider gitHubServiceProvider,
             IUsageService service,
-            IPackageSettings settings)
+            IPackageSettings settings,
+            JoinableTaskContext joinableTaskContext)
```

Comment:
Do we not intend to use `[Import(AllowDefault = true)]` here?

Reply:
I thought this as well before realizing that `UsageTracker` is a Visual Studio service not a MEF service (so `[Import(...)` isn't supported). We simply pass it the `JoinableTaskContext` from the host package.

Reply:
Ok, I understand.